.lintstagedrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "*.ts": ["prettier --write", "eslint"],
```

Comment:
run `npm run lint` instad of `eslint` to use previously configured eslint. use `--fix` for auth correct